avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,29 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, count, option=None, flood=False):
+        """
+        This method is used to check packet loss that occurs during
+        ping to peer address.
+
+        Function returns True for 0% packet loss and False
+        if packet loss occurs.
+
+        :param peer_ip: Peer IP address (IPv4 or IPv6)
+        :param count: How many packets to send
+        :param option: ping command options. Default is None
+        :param flood: ping flood option -f. Default is False
+        """
+        cmd = f'ping -I {self.name} {peer_ip} -c {count}'
+        if flood is True:
+            cmd = f'{cmd} -f'
+        elif option is not None:
+            cmd = f'{cmd} {option}'
+        try:
+            output = run_command(cmd, self.host)
+            if "0% packet loss" not in output:
+                return False
+            return True
+        except (NWException, IndexError):
+            raise NWException("ping Failed")
```

Comment:
Should be nice to expose details about the exception.

```python
except (NWException, IndexError) as ex:
    raise NWException("Ping failed") from ex
```

Reply:
Added {ex} message for the exception raised